From 07ff7baa79d23620d293b03fc2445c43bdacb34a Mon Sep 17 00:00:00 2001
From: James Le Cuirot <chewi@gentoo.org>
Date: Sat, 17 Feb 2024 14:37:44 +0000
Subject: [PATCH 12/14] rtpenc: Fix building with GCC 14

This incompatible pointer type issue became a fatal error in GCC 14. The
AVBuffer API started using size_t in 5.0 with ef6a9e5e.
---
 libavformat/rtpenc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavformat/rtpenc.c b/libavformat/rtpenc.c
index f67dc2a15a..1f1e4cb25a 100644
--- a/libavformat/rtpenc.c
+++ b/libavformat/rtpenc.c
@@ -588,7 +588,7 @@ static int rtp_write_packet(AVFormatContext *s1, AVPacket *pkt)
     case AV_CODEC_ID_H264:
     {
         uint8_t *side_data;
-        int side_data_size = 0;
+        size_t side_data_size = 0;
 
         side_data = av_packet_get_side_data(pkt, AV_PKT_DATA_NEW_EXTRADATA,
                                             &side_data_size);
-- 
2.34.1

